web/blog/arrow-data-spine/index.md

```diff
@@ -0,0 +1,128 @@
+row-oriented option worked well for this. But as security operations were
+maturing, requirements extended to analytical processing of structured data,
+making a columnar increasingly beneficial.
+
+Today, the need to bring advanced security analytics and data engineering
+together is stronger than ever, but there is a huge gap between the two fields.
+We see Arrow as the vehicle to close this gap, by practicing *security data
+engineering*. Security analytics should not only be easy and fast. It should
+also allow the expert to act in their domain, end-to-end. This requires with
+treating central domain objects as first-class (e.g., IP addresses, subnets, and
+URLs) and making working with them *easy*. This aspect is where our perception
+of Arrow changed: the primary value proposition of Arrow is to *make data
+interoperability easy*. With an open format specification at its core,
+Arrow-based runtimes were mushrooming all over the place. Nowadays it requires
+only a few lines of code to integrate Arrow data into the central logic of
+applications.
+
+Data interoperability is a sufficient, but not a necessary condition for
+enabling security analytics. We also need to embed the data model of the
+security domain. This is where Arrow's [extension types][extension-types] come
+into play. They add *semantics* to otherwise generic types, e.g., by telling the
+user "this is a transport-layer port" and not just a 16-bit unsigned integer, or
+"this is connection 4-tuple to represent a network flow" instead of "this is
+record with 4 fields of type string and unsigned integer". Extension types are
+composable and allow for creating a rich typing layer with meaningful domain
+objects on top of a standardized data representation. Since they are embedded in
+the data, they do not have to be made available out-of-band when crossing the
+boundaries of different tools. Now we have self-describing security data.
+
+Interoperability plus support for a domain-specific data model makes Arrow a
+solid *data plane*. But Arrow is much more than standardized data
+representation. Arrow also comes with bag of tools for working with the
+standardized data. In the diagram below, we show the various Arrow pieces that
+power the architecture of VAST:
+
+![Arrow Data Plane](arrow-data-plane.light.png#gh-light-mode-only)
+![Arrow Data Plane](arrow-data-plane.dark.png#gh-dark-mode-only)
+
+In the center we have highlighted the Arrow data plane that powers other
+parts of the system. Orange pieces highlight Arrow building blocks that we use
+today, and green pieces elements we plan to use in the future. There are several
+aspects worth pointing out:
+
+1. **Unified Data Representation**: When users ingest data into VAST, the
+   parsing process converts the native data into Arrow. Similarly, a
+   conversation boundary exists when data leaves the system, e.g., when a user
+   wants a query result shown in ASCII, JSON, CSV, or some custom format. Source
```

Comment:
```suggestion
   wants a query result shown in JSON, CSV, or some custom format. Source
```

What we call the `ascii` format is just a custom text-based representation without any use case other than debugging. I'd not mention it here.

Reply:
Removed `ASCII` all the way.